saleor/webhook/payloads.py

```diff
@@ -223,6 +226,61 @@ def generate_order_payload(order: "Order"):
     return order_data
 
 
+def calculate_added(
```

Comment:
When reading through this module, `calculate_added` and `calculate_removed` seem a bit unexpected names among all these payload generators, maybe we should make them "private"?

Reply:
Done.